vulnerabilities/tests/test_nvd.py

```diff
@@ -143,6 +145,13 @@ def test_to_advisories(self):
                         Reference(
                             url="http://kqueue.org/blog/2012/03/05/memory-allocator-security-revisited/",  # nopep8
                         ),
+                        Reference(
+                            url="https://nvd.nist.gov/vuln/detail/CVE-2005-4895",  # nopep8
+                            severities=[
+                                VulnerabilitySeverity(system=scoring_systems["cvssv2"], value="5.0")
```

Comment:
We need to be clear about what we store: the vector or the value? I thought we discussed storing the vector for now.
See https://github.com/nexB/vulnerablecode/pull/290/files#diff-d0d10e4267bd7b78581a8264137e41fad843af2fe6c813234a661e5a72930c2cR37
> "We store the vector as value and compute scores from that.",

and https://github.com/nexB/vulnerablecode/pull/290/files#diff-d0d10e4267bd7b78581a8264137e41fad843af2fe6c813234a661e5a72930c2cR17 to compute a score

Reply:
We will store both based on the new suggested design here, each as their own `VulnerabilitySeverity` and that for each Reference that provides these.